flang/runtime/stop.cpp

```diff
@@ -15,6 +15,7 @@
 #include <cfenv>
 #include <cstdio>
 #include <cstdlib>
+#include <execinfo.h>
```

Comment:
So far as I know this not a standard header - it is a GNU extension.

Flang supports other systems such as AIX, MacOS, Windows, and even other C libraries on gnu+linux systems (e.g. llvm-libc).

Please could you put this behind a preprocessor check so that compilation is not broken on those other systems.

Reply:
You're right, I'll change this asap.

Reply:
Hi, thank you very much for your suggestion. I have modified the code. You can check the latest commit. If possible, please give LGTM. Thank you.

Reply:
We have a config file with the results of the checks made by cmake: `<build-dir>/include/llvm/Config/config.h`.  One of these checks is for the `backtrace` function, and the header that declares it.
Please add
```
#include "llvm/Config/config.h"
```
then replace the `#ifdef __linux__` part with
```
#ifdef HAVE_BACKTRACE
#include BACKTRACE_HEADER
#endif
```